UniWatch/Views/Class/Index.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<UniWatch.Models.Class>
+
+<h3>My Classes</h3>
+
+<p>
+    @Html.ActionLink("New Class", "Create", new { teacherId = (int)ViewBag.TeacherId }, new { @class = "btn btn-success" })
+</p>
+
+<div class="table-responsive">
+    <table class="table table-bordered table-striped">
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Name)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Number)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Section)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Year)
+            </th>
+            <th></th>
+        </tr>
+
+        @foreach (var item in Model)
```

Comment:
Check if there are classes to display. Inform the user if there are not any. Something like the following:

```
@if(Model.Any())
{
  @foreach(...)
  { ... }
}
else
{
  <h4>There are no classes for this teacher</h4> 
}
```